elyra/util/operation_parser.py

```diff
@@ -0,0 +1,187 @@
+#
```

Comment:
I would split this into the handler and the actual service (two files) and this should be outside of util.